examples/evpn/calico/prepare.sh

```
@@ -2,11 +2,12 @@
 set -euo pipefail
 set -x
 CURRENT_PATH=$(dirname "$0")
+REPO_ROOT=$(git -C "${CURRENT_PATH}" rev-parse --show-toplevel)
 
 source "${CURRENT_PATH}/../../common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../common.sh was not specified as input (see shellcheck -x).

(SC1091)